src/app/core.py

```diff
@@ -206,13 +208,21 @@ def ntia_check_helper(request):
                                    )
             filename = fs.save(myfile.name, myfile)
             uploaded_file_url = fs.url(filename).replace("%20", " ")
-            """ Call the python function with parameters """
-            retval = check_anything.check_minimum_elements(str(settings.APP_DIR + uploaded_file_url)).messages
-            if (len(retval) > 0):
+            """ Call the python SBOM Checker """
+            schecker = SbomChecker(str(settings.APP_DIR + uploaded_file_url))
+            oldStdout = sys.stdout
+            tempstdout = StringIO()
```

Comment:
This `StringIO()` functionality could be the source of the unintended wrapping behavior. I'll investigate.

Nice work on this, PR! We'll get the output looking better next release :)